WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController+InformativeDialog.swift

```diff
@@ -15,10 +15,10 @@ extension GutenbergViewController {
     enum InfoDialog {
         static let key = "Gutenberg.InformativeDialog"
         static let message = NSLocalizedString(
-            "This post was originally created in the Block Editor, so we've also enabled it on this Post. Switch back to Classic at any time by tapping ••• in the top bar.",
-            comment: "Popup content about why this post is being opened in Block Editor"
+            "This post was originally created in the block editor, so we've also enabled it on this Post. Switch back to the classic editor at any time by tapping ••• in the top bar.",
+            comment: "Popup content about why this post is being opened in block editor"
         )
-        static let title = NSLocalizedString("Block Editor Enabled", comment: "Popup title about why this post is being opened in Block Editor")
+        static let title = NSLocalizedString("Block Editor Enabled", comment: "Popup title about why this post is being opened in block editor")
```

Comment:
Should this be "Block editor enabled"?

Reply:
I think so, yes. We use sentence case as a default.

Reply:
Thanks @yaelirub & @kristastevens updating coming in